webapp/index-self-contained.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>GeometryApp</title>
+
+		<script id="sap-ui-bootstrap"
+			src="resources/sap-ui-custom.js"
```

Comment:
would be nice to provide two build scripts in in the package.json.
One for self-contained and one for normal build